src/app/map-cases/map-cases.component.ts

```diff
@@ -30,51 +46,41 @@ export class MapCasesComponent implements OnInit, AfterViewInit {
     public dataSets = ['Infected', 'Recovered', 'Deaths'];
     public brushes = [
         [
-            'rgba(252, 32, 32, 0.3)',
-            'rgba(252, 32, 32, 0.55)',
-            'rgba(252, 32, 32, 0.8)',
-            'rgba(252, 32, 32, 0.99)',
+            'rgba(0,153,255, .3)',
+            'rgba(0,153,255, .5)',
+            'rgba(0,153,255, .7)',
+            'rgba(0,153,255, .95)',
         ],
         [
-            'rgba(50,205,50, 0.3)',
-            'rgba(50,205,50, 0.7)',
-            'rgba(50,205,50, 0.99)'],
+            'rgba(50,205,50, .3)',
+            'rgba(50,205,50, .7)',
+            'rgba(50,205,50, .95)'],
         [
-            'rgba(255, 0, 0, .5)',
-            'rgba(255, 0, 0, .8)',
-            'rgba(255, 0, 0, .99)']
+            'rgba(255, 0, 0, .3)',
+            'rgba(255, 0, 0, .7)',
+            'rgba(255, 0, 0, .95)']
     ];
     public data: string;
     public tooltipTitle = 'Infected';
     public showMap = true;
     private dataRequest$: any;
-
-    constructor(private dataService: RemoteDataService) {
-        this.dataSetButtons = [
-            {
-                name: 'Total',
-                selected: true
-            },
-            {
-                name: 'Recovered',
-                selected: false
-            },
-            {
-                name: 'Deaths',
-                selected: false
-            }
-    ];
-    }
-
     public ngOnInit(): void {
         this.loadDataSet(0);
     }
 
-    public ngAfterViewInit() {
-        const tileSource = new ArcGISOnlineMapImagery();
-        (tileSource as any).i = tileSource;
+    public changeMap(mapStyle) {
```

Comment:
rename mapStyle to soemthing that reveals more about the argument, in this case maybe darkThemeSelected, or darkTheme, and type it:
public changeMap(darkTheme: boolean).